easybuild/easyconfigs/g/GST-plugins-base/GST-plugins-base-1.8.3-foss-2016a.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'ConfigureMake'
+
+name = 'GST-plugins-base'
+version = '1.8.3'
+
+homepage = 'http://gstreamer.freedesktop.org/'
+description = """GStreamer is a library for constructing graphs of media-handling
+ components. The applications it supports range from simple
+ Ogg/Vorbis playback, audio/video streaming to complex audio
+ (mixing) and video (non-linear editing) processing."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['http://gstreamer.freedesktop.org/src/gst-plugins-base']
+sources = [SOURCELOWER_TAR_XZ]
+
+dependencies = [
+    ('GStreamer', '1.8.3'),
+    ('gettext', '0.19.7'),
+]
+
+builddependencies = [
+    ('Autotools', '20150215'),
```

Comment:
@boegel it was not in the requirements file, but I noticed it is used in order to build.